tests/languages/http/request-line_feature.test

```diff
@@ -12,48 +12,56 @@ GET / HTTP/1.1
 ----------------------------------------------------
 
 [
-	["request-line", [
-		["property", "POST"],
-		" http://example.com HTTP/1.0"
-	]],
-	["request-line", [
-		["property", "GET"],
-		" http://localhost",
-		["attr-name", ":9999"],
-		"/foo.html HTTP/1.1"
-	]],
-	["request-line", [
-		["property", "PUT"],
-		" http://www.example.com HTTP/2.0"
-	]],
-	["request-line", [
-		["property", "DELETE"],
-		" https://example.com HTTP/1.1"
-	]],
-	["request-line", [
-		["property", "OPTIONS"],
-		" https://www.example.com HTTP/1.1"
-	]],
-	["request-line", [
-		["property", "PATCH"],
-		" http://example.com HTTP/1.0"
-	]],
-	["request-line", [
-		["property", "TRACE"],
-		" http://example.com HTTP/1.0"
-	]],
-	["request-line", [
-		["property", "CONNECT"],
-		" http://example.com HTTP/1.0"
-	]],
-	["request-line", [
-		["property", "GET"],
-		" /path/to/foo.html HTTP/1.1"
-	]],
-	["request-line", [
-		["property", "GET"],
-		" / HTTP/1.1"
-	]]
+  ["request-line", [
+    ["method","POST"],

```

Comment:
Please use tabs for indentation.

Hint: We have a command to insert this JSON for you. If you remove the JSON between the horizontal lines and run `npm run test:languages -- --accept`, the current Prism output JSON will be inserted. More details can be found [here](https://prismjs.com/test-suite.html#writing-tests-writing-your-test).

Reply:
thanks, regenerated and confirmed.